other/two_sum.py

```diff
@@ -11,21 +11,36 @@
 Because nums[0] + nums[1] = 2 + 7 = 9,
 return [0, 1].
 """
+from typing import List
 
 
-def twoSum(nums, target):
+def two_sum(nums: List[int], target: int) -> List[int]:
```

Comment:
```suggestion
def two_sum(nums: list[int], target: int) -> list[int]:
```

Reply:
Travis tests fail without this change.

Reply:
I clicked "Commit suggestion" but I guess it didn't work.

Reply:
I committed it manually below, still waiting for Travis to kick in.

Reply:
You can `Add suggestion to batch` on all suggestions that then they will commit all at once.

Reply:
Oh! Didn't know that! Thanks. Github is so confusing! It should be good now.